python/chronos/src/bigdl/chronos/pytorch/utils.py

```diff
@@ -50,12 +69,24 @@ def _pytorch_fashion_inference(model, input_data, batch_size=None):
 
     :return: numpy ndarray
     '''
-    with torch.inference_mode():
-        if isinstance(input_data, list):
-            input_sample_list = list(map(lambda x: torch.from_numpy(x), input_data))
-        elif isinstance(input_data, DataLoader):
-            input_sample_list = [torch.cat(tuple(batch[0] for batch in input_data), dim=0)]
-        else:
-            input_sample_list = [torch.from_numpy(input_data)]
-        yhat = _inference(model, input_sample_list, batch_size=batch_size)
+    if hasattr(model, "output_tensors") and not model.output_tensors:
```

Comment:
should we add a `output_tensor` parameter and default to True for this `_pytorch_fashion_inference`? `_pytorch_fashion_inference` is an internal function so we could always call the correct `output_tensor` (maybe by maintain a `self.optimized_model_output_tensors`.

only check 1 bool could be much faster than 2 attr access (i.e., `hasattr` and `model.output_tensors`)